### PR TITLE
Added long callback and layout function examples

### PR DESCRIPTION
**Pull request by AnnMarieW**
Here are two more examples for the docs

### CODE REVIEW

File: docs/11-MultiPageDashApp-LayoutFunctions.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+```
+
+The reason `sidebar` and the layouts for the three topic pages need to be functions is that data is added to
+`dash.register_page`as each module is imported from the `pages` folder and `dash.register_page` is called.
+If you don't use a function then all the pages may not yet be in `dash.page_registry` when the sidebar is created.
+When you use a function, the layout is created when it's used rather than when it's imported.
+
+Here is what the app looks like:
+
+![layout_functions](https://user-images.githubusercontent.com/72614349/147510416-3529dabd-6cf4-4e4f-b7a6-027267778b88.gif)
```
Comment (Coding-with-Adam) [inline docs/11-MultiPageDashApp-LayoutFunctions.md:134]: Would it make sense to move this to the very top part of the page? I think the reader would appreciate being able to see what this chapter will teach them to create before they jump into all the reading.

File: docs/10-MultiPageDashApp-MetaTags.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ One of the nice features of this API is that it automatically creates the meta
 tags used by social media sites like Facebook and Twitter.  These sites use the app title, description and image to create
 the card that is displayed when you share a link to your site. They are also used in search engine results.
 
-Here is more info on social media meta tags:  https://css-tricks.com/essential-meta-tags-social-media/
```
[the remark this answers is not in the document]
Comment (AnnMarieW) [reply docs/10-MultiPageDashApp-MetaTags.md:18]: Good point.  No harm in including both references.

File: docs/11-MultiPageDashApp-LayoutFunctions.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+    [navbar, dl.plugins.page_container],
+    fluid=True,
+)
+
+if __name__ == "__main__":
+    app.run_server(debug=True)
+
+```
+
+This is the `home.py` file (The `about.py` file is similar).  We are taking advantage of the ability to
```
[the remark this answers is not in the document]
Comment (AnnMarieW) [reply docs/11-MultiPageDashApp-LayoutFunctions.md:61]: Ok, I'll elaborate on that.  I wasn't sure how much detail to go into on that.  Topic 1 is both the first topic in the side bar and is in the top navbar - so it acts as a landing page for the "topics" section.  I'm really seeing the value of being able to add  arbitrary props like "top_nav" to the `dash.page_registry`  In this case, it makes it easy to designate apps to one or more nav menus

File: docs/11-MultiPageDashApp-LayoutFunctions.md (hunk 1 above)
Comment (AnnMarieW) [reply docs/11-MultiPageDashApp-LayoutFunctions.md:134]: Oh, that would be a huge improvement --I'll make that change.  Thanks for your awesome feedback :-)

**Pull request merged**